File: backend/models/CauHinhTrangChu.js
const { DataTypes } = require("sequelize");
const sequelize = require("../config/db");

const CauHinhTrangChu = sequelize.define(
  "CauHinhTrangChu",
  {
    id: {
      type: DataTypes.INTEGER,
      primaryKey: true,
      autoIncrement: true,
    },
    ten_phan: {
      type: DataTypes.STRING,
      allowNull: false,
    },
    danh_muc_id_1: {
      type: DataTypes.INTEGER,
      allowNull: true,
    },
    ten_tab_1: {
      type: DataTypes.STRING,
      allowNull: true,
    },
    danh_muc_id_2: {
      type: DataTypes.INTEGER,
      allowNull: true,
    },
    ten_tab_2: {
      type: DataTypes.STRING,
      allowNull: true,
    },
    loai_hien_thi: {
      type: DataTypes.STRING(50),
      defaultValue: "ProductSection",
    },
    thu_tu: {
      type: DataTypes.INTEGER,
      defaultValue: 0,
    },
    du_lieu_json: {
      type: DataTypes.TEXT, 
      allowNull: true,
    },
    trang_thai: {
      type: DataTypes.STRING(20),
      defaultValue: "active",
    },
  },
  {
    tableName: "CauHinhTrangChu",
    timestamps: false,
  }
);

module.exports = CauHinhTrangChu;

File: backend-csharp/Models/CauHinhTrangChu.cs
using System;
using System.Collections.Generic;

namespace WebFashion.Api.Models;

public partial class CauHinhTrangChu
{
    public int Id { get; set; }

    public string TenPhan { get; set; } = null!;

    public int? DanhMucId1 { get; set; }

    public int? DanhMucId2 { get; set; }

    public string? LoaiHienThi { get; set; }

    public int? ThuTu { get; set; }

    public string? TrangThai { get; set; }

    public string? DuLieuJson { get; set; }

    public string? TenTab1 { get; set; }

    public string? TenTab2 { get; set; }
}
